<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>83</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="v1Label">
       <property name="text">
        <string>MyVector&lt;int&gt; v1:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="v1Elements">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="v1maxLabel">
         <property name="text">
          <string>max_size:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="v1MaxSize">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="v2Label">
       <property name="text">
        <string>MyVector&lt;int&gt; v2:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="v2Elements">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="v2MaxLabel">
         <property name="text">
          <string>max_size:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="v2MaxSize">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>791</width>
      <height>162</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="pushValue">
         <property name="placeholderText">
          <string>push_back value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="v1Push">
         <property name="text">
          <string>push_back to v1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="v2Push">
         <property name="text">
          <string>push_back to v2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="v1Pop">
         <property name="text">
          <string>pop_back v1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="v2Pop">
         <property name="text">
          <string>pop_back v2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="swapVec">
         <property name="text">
          <string>swap vectors</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="v1Clear">
         <property name="text">
          <string>clear v1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="v2Clear">
         <property name="text">
          <string>clear v2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLineEdit" name="resizeValue">
         <property name="placeholderText">
          <string>resize value</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="v1Resize">
         <property name="text">
          <string>resize v1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="v2Resize">
         <property name="text">
          <string>resize v2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLineEdit" name="atValue">
         <property name="placeholderText">
          <string>at index</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="v1At">
         <property name="text">
          <string>at v1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="v2At">
         <property name="text">
          <string>at v2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
